tensorflow_lite_support/cc/task/audio/BUILD

```diff
@@ -57,7 +57,7 @@ cc_library_with_tflite(
         "//tensorflow_lite_support/cc/port:status_macros",
         "//tensorflow_lite_support/cc/port:statusor",
         "//tensorflow_lite_support/cc/task/audio/proto:audio_embedder_options_cc_proto",
-        "//tensorflow_lite_support/cc/task/processor/proto:embedding_cc_proto",
+        "//tensorflow_lite_support/cc/task/processor/proto:embeddings_cc_proto",
```

Comment:
@kinaryml This may create breaking changes in projects which depend on this proto so please don't change the name. Meanwhile, I'm discussing with Maxime from TFLite engineering team to see if we should update the image_embedder API to use //tensorflow_lite_support/cc/task/processor/proto:embeddings_cc_proto instead of the embeddings proto in the vision folder.

Reply:
@khanhlvg The Text Embedder API seems to be using the embeddings proto from the Vision APIs too instead of `cc/task/processor/proto/embedding.proto` [here](https://github.com/khanhlvg/tflite-support/blob/master/tensorflow_lite_support/python/task/processor/proto/BUILD#L11)
So unless I change it to `embeddings` I think the tests might fail. Do we leave it as is for now?

Reply:
The [Text Embedder API](https://github.com/tensorflow/tflite-support/blob/4e3f799d0181d536845551f8c6f6b48a60657249/tensorflow_lite_support/cc/task/text/BUILD#L128) is actually using the `cc/task/processor/proto/embedding.proto`.

It isn't clear to me why keeping the proto as-is would cause the test to fail. Could you clarify?

Reply:
@khanhlvg It's because the Text Embedder Python API seems to be relying on [this](https://github.com/khanhlvg/tflite-support/blob/master/tensorflow_lite_support/python/task/text/text_embedder.py#L22) proto dependency (which uses the Vision API [embeddings](https://github.com/khanhlvg/tflite-support/blob/master/tensorflow_lite_support/python/task/processor/proto/BUILD#L11) proto)

The import would [here](https://github.com/khanhlvg/tflite-support/blob/master/tensorflow_lite_support/python/task/processor/proto/embeddings_pb2.py#L16) have to be changed to `from tensorflow_lite_support.cc.task.vision.proto import embedding_pb2` if task/processor/embedding.proto were to be used.

Reply:
I see. It looks like the TextEmbedder API is using different protos for the Python and C API.
* Python API: using `cc/task/vision/proto/embeddings.proto` as in [here](https://github.com/khanhlvg/tflite-support/blob/b57cdecd510daf87bc912676d1f2f9f39fca8cc5/tensorflow_lite_support/python/task/text/text_embedder.py#L22) and [here](https://github.com/khanhlvg/tflite-support/blob/master/tensorflow_lite_support/python/task/processor/proto/embeddings_pb2.py#L16)
* C++ API: using `cc/task/processor/proto/embedding.proto` as in [here](https://github.com/khanhlvg/tflite-support/blob/b57cdecd510daf87bc912676d1f2f9f39fca8cc5/tensorflow_lite_support/cc/task/text/text_embedder.h#L28).

But somehow the test still pass: `bazel test //tensorflow_lite_support/python/test/task/text:text_embedder_test`.

Therefore, I assume that you can follow the same pattern as in the text_embedder API.

@mbrenon @ziyeqinghan Is this the correct way to proceed?

Reply:
On a closer investigation of the code, I found that using different proto files will only cause issue if the proto is used as function parameter, such as in the case of image_embedder's [get_embedding_by_index](https://github.com/tensorflow/tflite-support/blob/a9dc5cb9a8f0e37a3cb82ed6956a5621b12468cf/tensorflow_lite_support/python/task/vision/image_embedder.py#L142). TextEmbedder Python API doesn't expose this method. 

If it's the return type of a function, Python doesn't have type check so it'll just work as long as the protos are identical.